airbyte-integrations/connectors/source-redshift/src/main/java/io/airbyte/integrations/source/redshift/RedshiftSource.java

```diff
@@ -39,7 +44,20 @@ public JsonNode toDatabaseConfig(final JsonNode redshiftConfig) {
             redshiftConfig.get("host").asText(),
             redshiftConfig.get("port").asText(),
             redshiftConfig.get("database").asText()));
+
+    if (redshiftConfig.has(SCHEMAS) && redshiftConfig.get(SCHEMAS).isArray()) {
+      schemas = new ArrayList<>();
+      for (final JsonNode schema : redshiftConfig.get(SCHEMAS)) {
+        schemas.add(schema.asText());
+      }
+    }
+
+    if (schemas != null && !schemas.isEmpty()) {
```

Comment:
can this be moved into the `if (redshiftConfig.has(SCHEMAS) && redshiftConfig.get(SCHEMAS).isArray())` block?

Reply:
Updated, thanks